frontend/src/App.js

```diff
@@ -1,9 +1,38 @@
-import React from 'react'
+import React from 'react';
+import { Provider } from 'react-redux';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import { user } from './reducer/user';
+
+import { LoginForm } from './components/LoginForm';
+import { CreateUser } from './components/CreateUser';
+import { UserPage } from './components/UserPage';
+import { Container } from './lib/Container';
+
+const reducer = combineReducers({ 
+  user: user.reducer,
+ });
+
+const store = configureStore({ reducer });
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <Provider store={store}>
+      <BrowserRouter>
+        <Switch>
+          <Route exact path="/">
+            <Container>
+              <LoginForm />
+              <CreateUser>
+                {/* <UserStatus /> */}
+                </CreateUser>
+            </Container>
+          </Route>
+          <Route path="/:id/user">
```

Comment:
I think the better solution is to make your user component get access to the id by param or from the reducer.  🤓